skymp5-client/src/front/browser.ts

```diff
@@ -1,93 +1,102 @@
 import {
-  browser,
-  on,
-  once,
-  Input,
-  printConsole,
-  settings,
-  Ui,
+	browser,
```

Comment:
Pls return spaces here

Reply:
ok, but i prefer 2 sizes tabs. And it seems to me that it is more correct.